ComputerInterface/Views/GameSettings/ItemSettingsView.cs

```diff
@@ -24,54 +23,51 @@ public override void OnShow(object[] args)
             Redraw();
         }
 
-		void UpdateState()
-		{
-			_selectionHandler.CurrentSelectionIndex = BaseGameInterface.GetItemMode() ? 1 : 0;
-            _insVolumeFloat = BaseGameInterface.GetInstrumentVolume();
-		}
+		void UpdateState() => _insVolumeFloat = BaseGameInterface.GetInstrumentVolume();
```

Comment:
Getting rid of the first line prevents the selection from being set to the correct value when the view is first shown. It will always show `Enabled`, even if it is `Disabled`.